system/updated/updated.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+import os
+import psutil
+import requests
+import shutil
+import time
+
+from pathlib import Path
+
+from openpilot.common.basedir import BASEDIR
+from openpilot.common.run import run_cmd
+from openpilot.common.params import Params
+from openpilot.common.realtime import set_core_affinity
+from openpilot.common.swaglog import cloudlog
+from openpilot.system.updated.casync import casync
+from openpilot.system.hardware import AGNOS
+from openpilot.system.updated.common import get_consistent_flag, set_consistent_flag
+from openpilot.system.version import BuildMetadata, get_build_metadata, build_metadata_from_dict
+
+from openpilot.system.hardware.tici.agnos import flash_partition, get_target_slot_number
```

Comment:
why is there still an agnos.py? the only agnos-specific things should be in the manifest right? applying that manifest should be totally generic

Reply:
made it more generic with `HARDWARE` to get the partition paths